src/utils/acl-mp-compat.lisp

```diff
@@ -51,21 +51,11 @@
   `(mp:with-lock (,lock)
     ,@body))
 
-#+(not (or sbcl openmcl cmu))
-(error "missing port for this compiler, please provide for multiprocessing primitives for this compiler in ~A" *load-pathname*)
-
 (defun make-process (function &key name)
-  #+sbcl (sb-thread:make-thread function :name name)
-  #+openmcl (ccl:process-run-function name function)
-  #+cmu (mp:make-process function :name name))
+  (bt:make-thread function :name name))
```

Comment:
the system already depended on bordeaux-threads, so I wasn't sure why it needed to reimplement the wheel. Let me know if you prefer me to just add the lispworks methods directly